### PR TITLE
Ingested MT Summit 1991.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/90/MTS-1991-TOC.htm for original.

### CODE REVIEW

**Comment by davidstap** [comment]
I just realised that this is the first MT Summit with proper abstracts, so it might be nice to add these.

File: data/xml/1991.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      <author><first>A.G.</first><last>Hauptmann</last></author>
+      <author><first>J.</first><last>Tebelskis</last></author>
+      <pages>113-117</pages>
+      <url hash="55c11554">1991.mtsummit-papers.18</url>
+      <abstract>We present JANUS, a speech-to-speech translation system that utilizes diverse processing strategies including connectionist learning, traditional AI knowledge representation approaches, dynamic programming, and stochastic techniques. JANUS translates continuously spoken English utterances into Japanese and German speech utterances. The overall system performance on a corpus of conference registration conversations is 87%. Two versions of JANUS are compared: one using an LR parser (JANUS-LR) and one using a neural-network based parser (JANUS-NN). Performance results are mixed, with JANUS-LR deriving benefit from a tighter language model and JANUS-NN benefiting from greater flexibility.</abstract>
+    </paper>
+  </volume>
+  <volume id="panels" ingest-date="2021-06-30">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit III</booktitle>
```
[the remark this answers is not in the document]
Comment (davidstap) [reply data/xml/1991.mtsummit.xml]: I think it's nice to at least split the panels. I gave the book titles different names. Do you think it is okay like this?

[the remark this answers is not in the document]
Comment (mjpost) [reply data/xml/1991.mtsummit.xml]: Yes, I agree it’s good to split entries related to the event itself (panels, invited talks) from the actual proceedings (written papers). So this looks good, thanks!

[the remark this answers is not in the document]
Comment (davidstap) [reply data/xml/1991.mtsummit.xml]: Then this seems ready to merge? Can you approve? :-)

**Review by mjpost — APPROVED** [review]

**Pull request merged**